src/test.rs

```diff
@@ -14,239 +14,237 @@ macro_rules! p {
     }
 }
 
-#[test]
-fn connect_google() {
-    let builder = p!(TlsConnector::builder());
-    let builder = p!(builder.build());
-    let s = p!(TcpStream::connect("google.com:443"));
-    let mut socket = p!(builder.connect("google.com", s));
-
-    p!(socket.write_all(b"GET / HTTP/1.0\r\n\r\n"));
-    let mut result = vec![];
-    p!(socket.read_to_end(&mut result));
-
-    println!("{}", String::from_utf8_lossy(&result));
-    assert!(result.starts_with(b"HTTP/1.0"));
-    assert!(result.ends_with(b"</HTML>\r\n") || result.ends_with(b"</html>"));
-}
+mod tests {
```

Comment:
rust_test_ios requires the tests to be indented, and this seemed like the most reliable way. I tried to remove the requirement but it's a substantial piece of work.